tests/resources/VCH-Util.robot

```diff
@@ -159,6 +159,22 @@ Check For The Proper Log Files
     Should Contain  ${output}  ${container}/vmware.log
     Should Contain  ${output}  ${container}/tether.debug
 
+Scrape Logs For the Password
```

Comment:
Can we add a Docker operation that actually does a thing against vSphere after login but before we do the scrape for the password so that we can make sure that the password doesn't leak out in other logs? There really shouldn't be a way for it to, because it shouldn't ever be passed to those components in the first place as they have a token instead, but this would allow us ongoing assurance that it does not.